lib/_debugger.js

```diff
@@ -4,8 +4,8 @@ const util = require('util');
 const path = require('path');
 const net = require('net');
 const vm = require('vm');
-const module = require('module');
-const repl = module.requireRepl();
+const Module = require('module');
```

Comment:
@bnoordhuis are you OK with the capitalization? If not, we could replace the two following instances with `require('module')`, use a different name, or switch from `const`. Any preference?